xbmc/settings/dialogs/GUIDialogSettingsManualBase.cpp

```diff
@@ -239,6 +245,24 @@ CSettingAction* CGUIDialogSettingsManualBase::AddButton(CSettingGroup *group, co
   return setting;
 }
 
+CSettingString* CGUIDialogSettingsManualBase::AddInfoStringButton(CSettingGroup *group, const std::string &id, int label, int level, std::string info, bool delayed /* = false */,
```

Comment:
Thought if it is possible to add, the <code>delayed</code>, but becomes currently in DSP not used.

The function is used to show the cpu usage for DSP process and if present to open addon dialog.